addons/crm_partner_assign/wizard/crm_forward_to_partner.py

```diff
@@ -112,73 +94,56 @@ def action_forward(self, cr, uid, ids, context=None):
             local_context['partner_id'] = partner_leads['partner']
             local_context['partner_leads'] = partner_leads['leads']
             local_context['partner_in_portal'] = in_portal
-            email_template_obj.send_mail(cr, uid, template_id, ids[0], context=local_context)
+            lead_template.with_context(local_context).send_mail(lead_template.id)
             lead_ids = [lead['lead_id'].id for lead in partner_leads['leads']]
```

Comment:
Why not lead['lead_id'] only and use it at line 102 and 103(no need to re browse)